network-store-server/src/main/java/com/powsybl/network/store/server/NetworkStoreRepository.java

```diff
@@ -27,6 +25,7 @@
 import java.sql.SQLException;
 import java.util.*;
 import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
 
 import static com.powsybl.network.store.server.QueryBuilder.*;
```

Comment:
QueryBuilder could be removed?

Reply:
no, but cleaned?

Reply:
I made some cleaning in this class

Using a specific API to construct the queries (like jooq) can be the object of another US